spec/rollbar/plugins/thread_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+Rollbar.plugins.load!
+
+describe Rollbar::ThreadPlugin do
```

Comment:
There was no spec specifically for this plugin, so I thought I'd add this to improve coverage.

Reply:
Thank you!